pyaedt/maxwell.py

```diff
@@ -548,13 +567,18 @@ def assign_current(self, object_list, amplitude=1, phase="0deg", solid=True, swa
         amplitude : float or str, optional
             Current amplitude. The default is ``1A``.
         phase : str, optional
+            Current phase.
             The default is ``"0deg"``.
         solid : bool, optional
-            The default is ``True``.
+            Specify the type of conductor.
+            It can be ``Solid`` or ``Stranded``.
+            The default is ``True`` which means ``solid``, in the other case it means ``stranded``.
         swap_direction : bool, optional
+            Reference direction.
             The default is ``False``.
```

Comment:
This should be described like the previous parameter. What does "False" versus "True" mean in respect to reference direction?